daemon/logger/awslogs/cloudwatchlogs_test.go

```diff
@@ -641,14 +641,28 @@ func TestCollectBatchMultilinePatternMaxEventAge(t *testing.T) {
 	})
 
 	// Fire ticker batchPublishFrequency seconds later
-	ticks <- time.Now().Add(batchPublishFrequency * time.Second)
+	ticks <- time.Now().Add(batchPublishFrequency + time.Second)
```

Comment:
Is this correct? The comment leads me to believe it should be "* time.Second";

     // Fire ticker batchPublishFrequency seconds later